qrenderdoc/CMakeLists.txt

```diff
@@ -115,14 +115,15 @@ include(ExternalProject)
 # Need bison for swig
 find_package(BISON)
 
-set(SWIG_CONFIGURE_CC ${CMAKE_C_COMPILER})
-set(SWIG_CONFIGURE_CXX ${CMAKE_CXX_COMPILER})
-
+set(SWIG_CONFIGURE_CC_DEFAULT ${CMAKE_C_COMPILER})
```

Comment:
Out of interest does the variable name have to change and does the variable have to be cached?

It is hard to diagnose the change without being able to replicate the problem

Reply:
It doesn't have to be cached - it just seemed a better fit for a variable that's unlikely to change between CMake runs - however I'll update it in the spirit of keeping the changes minimal.

Reply:
Thank you